app/src/main/java/org/fossasia/pslab/fragment/HomeFragment.java

```diff
@@ -33,11 +39,18 @@
     TextView tvVersion;
     @BindView(R.id.img_device_status)
     ImageView imgViewDeviceStatus;
+    @BindView(R.id.tv_device_description)
+    TextView deviceDescription;
+    @BindView(R.id.tv_connect_msg)
+    TextView tvConnectMsg ;
+    @BindView(R.id.pslab_web_view)
+    WebView webView;
     private Unbinder unbinder;
 
-    @BindView(R.id.tv_initialisation_status)
-    TextView tvInitializationStatus;
+//    @BindView(R.id.tv_initialisation_status)
+//    TextView tvInitializationStatus;
     public static InitializationVariable booleanVariable;
+    public static boolean web_view_is_showing=false;
```

Comment:
This is not a good naming convention for Java. You can take a look at [here](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#367)